index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="es" xml:lang="es">
```

Comment:
@edwartexe I see the content is in english, so you must use `en`
```suggestion
<html lang="en" xml:lang="en">
```